README.md

```diff
@@ -1,105 +1,7 @@
-- When writing an `obeys` rule, results will include `Invariant` names.
-
-## Go to Definition
-
-![FSH Go to Definition](images/docs/fsh-go-to-definition.gif)
-
-FSH entity definitions within your workspace can be found from anywhere their name is used. To go to the entity definition, right-click on the entity name and select "Go to Definition" from the context menu. Or, you can press &lt;F12&gt; when your text cursor is on the entity name.
-
-## Open FHIR Documentation
-
-![FSH Go to Definition](images/docs/fsh-open-fhir-documentation.gif)
-
-Documentation pages can be opened directly from your FSH files. Right-click on the name of any FHIR resource or FSH keyword, and select "Open FHIR Documentation" from the context menu.
-
-## Compile and Run (for Developers)
-
-- run `npm install` in this folder. This installs all necessary npm modules in both the
-  client and server folder
-- open VS Code on this folder.
-- Switch to the Debug viewlet.
-- Select `Extension` from the drop down.
-
-## NPM Tasks
-
-The following NPM tasks may be useful in development:
-| Task | Description |
-| ---- | ----------- |
-| **build** | compiles `src/**/*.ts` files to `out/**/*.js` files using the TypeScript compiler (tsc) |
-| **build:watch** | similar to _build_ but automatically builds when changes are detected in src files |
-| **lint** | checks all src files to ensure they follow project code styles and rules |
-| **lint:fix** | fixes lint errors when automatic fixes are available for them |
-| **prettier** | checks all src files to ensure they follow project formatting conventions |
-| **prettier:fix** | fixes prettier errors by rewriting files using project formatting conventions |
-| **test** | runs the test suite |
-| **check** | runs all the checks performed as part of ci (lint, prettier, test) |
-
-To run any of these tasks, use `npm run`. For example:
-
-```sh
-$ npm run build:watch
-```
-
-### Note about testing
-
-The `test` task will often (but not always) produce an error if it is run while VSCode is open. If you receive this error, close VSCode and run the task from the command line.
-
-## Updating the Grammar
-
-The `src/lang/` directory contains the FSH lexer and parser. When the FSH grammar changes, the files in this directory will need to be updated. These files are generated in the [SUSHI](https://github.com/FHIR/sushi) project by running ANTLR on the grammar definition files. The files are then copied from the SUSHI project to this project.
-The files to copy are:
-
-- FSH.tokens
-- FSHLexer.tokens
-- FSHLexer.js
-- FSHListener.js
-- FSHParser.js
-- FSHVisitor.js
+If you do already have this extension installed, it should automatically install the mitre-health.vscode-language-fsh version as a dependency once you are on version 1.7.0. Once the mitre-health.vscode-language-fsh extension is installed, feel free to uninstall this kmahalingam.vscode-language-fsh extension, since it is not functional.
```

Comment:
Since we don't know that the `extensionDepdnencies` trick will work yet, I don't necessarily think we should update the language here yet. But one question I do have about the approach is if the new MITRE Health extension will show up in the list of installed extensions in VS Code and if it will continue to auto-update when we publish new versions. If the extension doesn't show up in the list or doesn't auto-update, we might want to suggest here that users download the new extension regardless. I don't know that we know the answer to how this works before we publish, so I don't think you need to update the language now, but just a thought to consider after we know how this works.